packages/terra-i18n-plugin/src/I18nAggregatorPlugin.js

```diff
@@ -15,42 +14,48 @@ const messages = ${JSON.stringify(messages, null, 2)};
 module.exports = {
   areTranslationsLoaded: true,
   locale: '${language}',
-  messages,
+  messages: messages
```

Comment:
We will want to keep the shorthand notation, updating to `messages: messages` will result in eslint error `expected shorthand property`.

Reply:
It's to resolve this issue: https://github.com/cerner/terra-core/issues/878

Reply:
The shorthand syntax is what was causing issue #878, therefore changed it back to this.  I've ran the linter and didn't receive any errors as it is inside ``.

Reply:
^ The error would have been seen in the resulting file, however I just checked, the `.eslintignore` file actually ignores the aggregated-translations dir so this won't cause an issue either way.